website/addons/owncloud/serializer.py

```diff
@@ -0,0 +1,65 @@
+from website.addons.base.serializer import StorageAddonSerializer
+from website.addons.owncloud.settings import DEFAULT_HOSTS
+from website.util import web_url_for
+from website.addons.owncloud.utils import ExternalAccountConverter
```

Comment:
Woops! Sorry, I'm in the middle of making changes from the last set of comments, and I accidentally pushed. I'll finish up with the refactor quickly and push again when ready